check-code-style/check_style.sh

```diff
@@ -24,11 +24,21 @@ check_version() {
 # clang-format because we clang-format to respect programmer added
 # newlines in places where it makes the code easier to read than what
 # clang-format might have done itself.
+# To skip the length check for a line (e.g. for a long URL), put 
+# "check_line_length skip" in a comment on the previous line.
 check_line_length() {
   local file="${1}"
   local -i status=0
   local -i number=1
+  local -i skip_next_line=0
   while IFS= read -r line; do
+    if (( skip_next_line )); then
+      (( skip_next_line-- ))
```

Comment:
Can we use bools rather than counters? I find it confusing that we're decrementing a thing that can only be 1 or 0

https://stackoverflow.com/questions/2953646/how-can-i-declare-and-use-boolean-variables-in-a-shell-script

Reply:
we definitely can! I didn't know that was a thing